File: Cmentarz/Cmentarz/Models/Wlasciciel.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cmentarz.Models
{
    [Table("Wlasciciele")]
    public class Wlasciciel
    {
        [Key]
        public int IdWlasciciel { get; set; }
        [Required]
        public string Imie { get; set; }
        [Required]
        public string Nazwisko { get; set; }
        [Required]
        public string Adres { get; set; }
        public int IlGrobowcow { get; set; }
        [Required]
        public List<Grobowiec> Lista_Grobowcow { get; set; }
       
        public int IdUzytkownik { get; set; }
        public  Uzytkownik Uzytkownik { get; set; }
    }
}

File: Cmentarz/CmentarzAngular/src/app/models/wlasciciel.ts
import { Grobowiec } from "./grobowiec";

export interface Wlasciciel {
    idWlasciciel: number;
    imie: string;
    nazwisko: string;
    adres: string;
    ilGrobowcow: number;
    grobowce: Grobowiec[];
}
  